public/js/investment.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+class Investment {
+
+  constructor(symbol, share_price, num_shares) {
+    this.symbol = symbol;
+    this.share_price = share_price;
+    this.num_shares = num_shares;
+    this.total_purchase_price = share_price * num_shares;
+  }
+
+  to_dict() {
```

Comment:
can we add some comments explain what the method is for, and same for from_dict

Reply:
will add now!